Lib/ldap/schema/models.py

```diff
@@ -126,6 +126,9 @@ class ObjectClass(SchemaElement):
   sup
     This list of strings contains NAMEs or OIDs of object classes
     this object class is derived from
+  x-origin
+    This string contains the X-ORIGIN text which is typically used to indicate
+    the source of the associated schema element. It can a list of strings
```

Comment:
```suggestion
    the source of the associated schema element. It should be be a tuple of strings.
```

Reply:
Agree. But it has a small typo - "be be"

Reply:
Also, should we change other docstrings then for the consistency? For example, 'may' also should be a tuple but the docstring states 'This list of strings':

    may
        This list of strings contains NAMEs or OIDs of additional attributes
        an entry of the object class may have

Reply:
I guess "[sequence](https://docs.python.org/3/glossary.html#term-sequence)" is the exact thing we want here.
Perhaps write it out as "sequence (list/tuple)", to aid people who don't actively know that definition.

Reply:
Actually, list wouldn't work because the SchemaElement class explicitly requires a 'tuple'.

``` python
class SchemaElement:
    def key_list(self,key,values,sep=' ',quoted=0):
        assert type(values)==tuple,TypeError("values has to be a tuple, was %r" % values)
```